build.gradle

```diff
@@ -1,17 +1,20 @@
 buildscript {
+    apply from: rootProject.file("gradle/versions.gradle")
```

Comment:
I renamed it to "dependencies.gradle" but why not keep it in the "gradle" folder? You have "gradle-mvn-push.gradle" in there as well.